### PR TITLE
Fix inconsistencies on the frontend

### PR DESCRIPTION
**Pull request by mrica-equinor**

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx
Comment (andchiind) [inline frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx:35]: We can probably rename this to something other than a dialog since it’s not being used that way anymore

**Review by andchiind — COMMENTED** [review]
LGTM but it’s probably best to wait for another approval to merge since I can’t actually test it atm

**Comment by prasm313** [comment]
Looks good. Tested the improvements both in the ipad and computer view.

**Review by prasm313 — APPROVED** [review]
LGTM

**Pull request merged**
